internal/delete.go

```diff
@@ -0,0 +1,49 @@
+package internal
+
+import (
+	"fmt"
+	"os"
+)
+
+func DeleteApp(appName string) {
```

Comment:
Please add unit tests for this method. This will help ensure the quality and test coverage of the project. Thank you in advance for your contribution!